src/model.py

```diff
@@ -23,27 +37,24 @@ def __init__(self, scaler, num_nodes, lrate, wdecay, device, adj_mx):
         self.edge_index = [[], []]
         self.edge_weight = []
 
-        # The adjacency matrix is converted into an edge_index list
```

Comment:
instead of taking in adj matrix, changed to take in edge_index and edge_weight directly